src/plays/2048/Game2048Styles/Modal.css

```diff
@@ -0,0 +1,87 @@
+body.active-modal {
```

Comment:
@Shivam-Katare Please consider adding unique style names in this file too.

Reply:
Okay.